example/browser.js

```diff
@@ -0,0 +1,15 @@
+require('es6-promise').polyfill();
```

Comment:
In examples I try to slim things down to their minimum possible requirements.  And it doesn't look like you are using promises anywhere?  Also, this was for browser support, which we might be able to remove even in our apps, so we can remove them in the examples for sure.